rllib/algorithms/ppo/ppo_base_learner.py

```diff
@@ -14,6 +15,9 @@ def __init__(self, *args, **kwargs):
         if self.hps.entropy_coeff_schedule:
             raise ValueError("entropy_coeff_schedule is not supported in Learner yet")
 
+        # TODO (Kourosh): This needs to be native tensor variable to be traced.
```

Comment:
does it need to be traced? only if we need grads fed through it, which I dont think is the case

Reply:
oops i take it back